blueoil/converter/templates/include/func/cast.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2018 The Blueoil Authors. All Rights Reserved.
```

Comment:
The year of first publication.
```suggestion
/* Copyright 2020 The Blueoil Authors. All Rights Reserved.
```